app/controllers/navigation/Navigator.scala

```diff
@@ -406,16 +423,36 @@ class Navigator @Inject()(
       routes.CommodityDetailsController.displayPage(mode, itemId)
 
   private def additionalDocumentsSummaryPreviousPage(cacheModel: ExportsDeclaration, mode: Mode, itemId: String): Call =
-    if (cacheModel.itemBy(itemId).flatMap(_.additionalInformation).exists(_.items.nonEmpty))
+    routes.IsLicenseRequiredController.displayPage(mode, itemId)
+
+  private def additionalDocumentsPreviousPage(cacheModel: ExportsDeclaration, mode: Mode, itemId: String): Call = {
+
+    val isLicenseRequired = cacheModel.itemBy(itemId).exists(_.isLicenseRequired.contains(true))
+
+    if (cacheModel.isAuthCodeRequiringAdditionalDocuments || isLicenseRequired) routes.IsLicenseRequiredController.displayPage(mode, itemId)
+    else routes.AdditionalDocumentsRequiredController.displayPage(mode, itemId)
+
+  }
+
+  private def additionalDocumentsSummaryClearancePreviousPage(cacheModel: ExportsDeclaration, mode: Mode, itemId: String): Call =
+    if (cacheModel.listOfAdditionalInformationOfItem(itemId).nonEmpty)
       routes.AdditionalInformationController.displayPage(mode, itemId)
     else
       routes.AdditionalInformationRequiredController.displayPage(mode, itemId)
```

Comment:
Wouldn't the previous page from additional documents summary just be /additional-documentation?

Reply:
I'm maintaining the same logic as what came before this story here, so I'm hesitant to fiddle with the back button any further!